src/publish.js

```diff
@@ -84,10 +84,7 @@ module.exports = function strain() {
           'Fastly Service ID is required.',
         )
         .check((args) => {
-          if (args.githubToken && args.updateBotConfig === undefined) {
-            // eslint-disable-next-line no-param-reassign
-            args.updateBotConfig = true;
-          } else if (args.updateBotConfig && !args.githubToken) {
+          if (args.updateBotConfig && !args.githubToken) {
```

Comment:
hmm...you changed the behaviour for automatically updating the bot config... why?

Reply:
Because `githubToken` is not used exclusively for configuring the bot but also for supporting authenticated github access. The implicit bot configuration when providing a github token was just a convenience feature and is confusing now that the github token is also used for authenticated github access.

Reply:
I disagree. bot config should always be updated when a github token is present. the previous behaviour was correct and should be retained.